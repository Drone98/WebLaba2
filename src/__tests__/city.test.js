import renderer from "react-test-renderer";
import City from '../classes/city'
import React from "react";

const data = {
            "data": {
                "coord": {
                    "lon": 66.66,
                    "lat": 66.66
                },
                "weather": [
                    {
                        "description": "overcast clouds",
                        "icon": "04d"
                    }
                ],
                "main": {
                    "temp": 0,
                    "pressure": 1000,
                    "humidity": 80
                },
                "wind": {
                    "speed": 5
                }
            }
};

it('Rendered', () => {
    let component = renderer.create(
        <City data={data.data}/>
    );
    let cur = component.toJSON();
    expect(cur).toMatchSnapshot();
});
