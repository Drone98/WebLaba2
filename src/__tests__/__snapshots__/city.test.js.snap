// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendered 1`] = `
<ul>
  <li>
    <span>
      Ветер
    </span>
     
    <em>
      5
       м/c
    </em>
  </li>
  <li>
    <span>
      Облачность
    </span>
     
    <em>
      overcast clouds
    </em>
  </li>
  <li>
    <span>
      Давление
    </span>
     
    <em>
      750
       мм рт. ст.
    </em>
  </li>
  <li>
    <span>
      Влажность
    </span>
     
    <em>
      80
       %
    </em>
  </li>
  <li>
    <span>
      Координаты
    </span>
     
    <em>
      [
      66.66
      , 
      66.66
      ]
    </em>
  </li>
</ul>
`;
